TCAT/Assets.xcassets/live.imageset/Contents.json

```diff
@@ -0,0 +1,26 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "live.png",
```

Comment:
I added a color parameter in the `LiveIndicator` view. I'll have to add an enum or size parameters as well. We should stick to one use of this throughout so we're consistent with the look and animation. Although if this should be static and different, so be it. I'll ask Mihir

Reply:
Good point. Lmk when you add the other parameters and I can use that instead of the picture